src/styles/_mixins.scss

```diff
@@ -108,3 +108,15 @@
     @include non-visual-button-rules;
   }
 }
+
+@mixin visually-hidden {
```

Comment:
suggestion: can you add a comment here on when this should be used/what it does?

Technically an element using this mixin isn’t actually hidden since it takes up 1x1 pixels in the UI.